mirrord/intproxy/protocol/src/lib.rs

```diff
@@ -117,13 +118,49 @@ impl fmt::Display for NetProtocol {
     }
 }
 
+bitflags! {
```

Comment:
I don't think int proxy should know whether it's blocking or nonblocking.

Reply:
but.. getpeeraddr is hooked by us anyway, why not modify the struct/hook to support it?